docs/getting-started-sandbox.md

```diff
@@ -0,0 +1,91 @@
+---
+id: getting-started-sandbox
+title: Getting Started With PayID Sandbox
+sidebar_label: Getting Started With PayID Sandbox
+---
+
+With the [PayID Sandbox](https://payid.org/sandbox), you can experiment with PayID without deploying your own server. Although the PayID Sandbox can be used to create PayIDs, do not use these PayIDs that rely on this virtual server to send or receive real cryptocurrency. The PayID Sandbox does not directly support Verifiable PayID.
```

Comment:
> do not use these PayIDs that rely on this virtual server to send or receive real cryptocurrency.

This is probably worth a separate callout (a Caution, perhaps).

Reply:
Also suggest small tweak:

> do not use PayIDs that rely on this virtual server to send or receive real cryptocurrency.

(remove "these")

Reply:
Done